### DISCUSSION TITLE
guide/markdown/basic/

### DISCUSSION THREAD

**Discussion by giscus** [pengzhanbo/vuepress-theme-plume#201] [docs-comment]
# guide/markdown/basic/

注 此文档 Fork 自 vuepress-theme-hope, 遵循 MIT 许可证。 Markdown 是一种可以使用普通文本编辑器编写的标记语言，通过简单的标记语法，它可以使普通文本内容具有一定的格式。 Markdown 的目标是实现「易读易写」。 概述 不过最需要强调的便是它的可读性。一份使用 Markdown 格式撰写的文件应该可以直接以纯...

https://theme-plume.vuejs.press/guide/markdown/basic/

**Comment by mnjun9527** [comment]
請問如果我是否可以在 VITE中使用這個主題的某個部分,  比如 加密   或者圖表的嵌入, 我應該如何做才能實現呢?

**Reply by pengzhanbo** [reply]
请自行实现。

**Comment by QiHuang02** [comment]
请问四级以下的标题不会出现在`此页内容`部分吗？

**Reply by pengzhanbo** [reply]
查看 [outline 配置](https://theme-plume.vuejs.press/config/basic/#outline)

**Comment by zhengchen-zzZ** [comment]
【强调】在移动端不太明显

**Reply by pengzhanbo** [reply]
这与设备使用的字体有关，强调的字重为 600 ，一般这个值在可以理解为 bold ，如果设备使用的字体只有 Regular ，而不包含 Bold ，或者 字体的 Bold 区别不明显，那么 强调 可能在视觉上就会受到影响。

**Comment by Easton119** [comment]
请问下`~~xxx~~`删除语法为什么有时候生效，有时候无效呢。
比如下面这个就是没有生效的：
![image.png](https://fastly.jsdelivr.net/gh/easton119/oss/test111/20250203105637.png)

**Reply by pengzhanbo** [reply]
不要在 `~~` 的内边缘留有空格，它应该紧贴文本。

```md
文本 ~~文本~~ 文本      ✔️

文本 ~~ 文本~~ 文本     ❌
       ^

文本 ~~文本 ~~ 文本     ❌
           ^
```

**Reply by Easton119** [reply]
事实上我没有留空白，这是我的md文件：
![image](https://github.com/user-attachments/assets/5bfaaa3d-b449-46d0-9e8f-682375d94d7a)
这是网页上的效果：
![image](https://github.com/user-attachments/assets/02700906-4962-460d-bb4d-ecbff208e26a)
然后我发现如果我把括号去掉，又生效了：
![image](https://github.com/user-attachments/assets/9b77ad88-9f9b-489a-afbd-2c837467e200)
真是很奇怪🤔

**Reply by pengzhanbo** [reply]
在 `~~` 的外边缘添加一个空格即可解决。

**Comment by coding-er** [comment]
1211

**Comment by hub-yu** [comment]
作者您好:
我在markdown中使用多级无须列表，内容如下
```
- Red
  - Green
- Blue
```
此时第二级别列表确实发生了缩进，
但并非是空心点，而是和一级一样是实心点。
我不清楚这是一个BUG？还是设计就是如此。

**Comment by Mars13333** [comment]
大佬，你好。
关于目录的显示，能不能多显示一级别，现在好像只有H2和H3能在目录中显示。
俩级感觉太少了。

**Reply by pengzhanbo** [reply]
[theme config - outline](https://theme-plume.vuejs.press/config/theme/#outline)
